tests/sdp_primal_dual_test.cpp

```diff
@@ -182,8 +182,11 @@ ConstructMaxCutSDPFromLaplacian(const std::string& laplacianFilename)
 
 static bool CheckPositiveSemiDefinite(const arma::mat& X)
 {
-  const auto evals = arma::eig_sym(X);
-  return (evals(0) > 1e-20);
+  arma::vec evals;
+  if(!arma::eig_sym(evals, X))
```

Comment:
Just a minor style issue, missing space right after the `ìf``, can be fixed in the merge process.